pkg/utils/node/predicates.go

```diff
@@ -42,13 +47,27 @@ func IsPastTTL(node *v1.Node) bool {
 	return time.Now().After(ttlTime)
 }
 
-// TODO: implement underutilized function (some generalized predicate)
-func IsUnderutilized(pods []*v1.Pod) bool {
-	counter := 0
+// IsUnderutilized returns true if all of a node's pod's resource requests sum to less than the given thresholds
```

Comment:
I am not sure if we really should be doing this. How do we avoid getting in a loop? We might consider here a node as underutilized and then remove it, pods will get into pending state, allocator will bring this node back again. 
Are we checking if these pods can be rescheduled somewhere else?

Reply:
Currently, I haven't added that logic in. It's in a list of TODOs. For now, I wanted to add this generalized utilization (which we can tone down to 0%) until we have the necessary logic to make this work.